plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/connection/OAuthDetailView.java

```diff
@@ -212,6 +212,11 @@ public String toString() {
 		}
 
 		@Override
+		public void widgetDefaultSelected(SelectionEvent e) {
+			widgetDefaultSelected(e);
+		}
```

Comment:
the imho more relevant other tests I did (low-level SWT listeners etc.) are reported in my jira: https://issues.jboss.org/browse/JBIDE-20092?focusedCommentId=13084745&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13084745 (if you're interested)